packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -476,8 +476,13 @@ export function signInActor({ services }: SignInMachineOptions) {
               `${country_code}${phone_number}`.replace(/[^A-Z0-9+]/gi, '');
             rest = { ...rest, phone_number: phoneNumberWithCountryCode };
           }
+          try {
+            await Auth.completeNewPassword(user, password, rest);
+          } catch (err) {
+            return Promise.reject(err);
+          }
 
-          return Auth.completeNewPassword(user, password, rest);
+          return Auth.currentAuthenticatedUser();
```

Comment:
`Auth.currentAuthenticatedUser` returns `user` object with all the user attributes: https://docs.amplify.aws/lib/auth/manageusers/q/platform/js/#retrieve-attributes-for-current-authenticated-user